apps/omg_child_chain_rpc/test/omg_child_chain_rpc/web/plugs/health_test.exs

```diff
@@ -93,7 +101,10 @@ defmodule OMG.ChildChainRPC.Plugs.HealthTest do
   defp pull_client_alarm(0, _, _), do: :cant_match
 
   defp pull_client_alarm(n, match, fnn) do
-    case fnn.() do
+    endpoint_call_result = fnn.()
+    match = Map.put_new(match, "version", Map.get(endpoint_call_result, "version"))
```

Comment:
It would be better to get the key out.

Reply:
what do you mean key out?

Reply:
do you mean key from the `match` variable? it's no there, that's why we're putting it there :)

Reply:
why, the version comes as a response from the server? why would we want to match to the response?